Makefile

```diff
@@ -40,10 +40,26 @@ from-scratch: ## start development environment from scratch
 	make up
 
 devstack-install: ## install local version of package into docker devstack for development
-	docker exec -t edx.devstack.lms bash -c 'source /edx/app/edxapp/nodeenvs/edxapp/bin/activate && cd /edx/app/edxapp/edx-platform && npm uninstall @edx/studio-frontend && cd /edx/src/studio-frontend && npm link && cd /edx/app/edxapp/edx-platform && npm link @edx/studio-frontend'
+	# note that this assumes a structure /src/sfe_parent/studio-frontend
+	# step 0: remove studio-frontend from edx-platform/node_modules
+	docker exec -t edx.devstack.studio bash -c 'source /edx/app/edxapp/nodeenvs/edxapp/bin/activate && cd /edx/app/edxapp/edx-platform && npm uninstall @edx/studio-frontend && rm -rf ./node_modules/@edx/studio-frontend'
+	# step 1: create symlink in /edx/src/sfe_parent/symlinks
+	# note that we delete studio-frontend/node_modules; this is deliberate
+	docker exec -t edx.devstack.studio bash -c 'mkdir -p /edx/src/sfe_parent/symlinks && rm -rf /edx/src/sfe_parent/symlinks/* && source /edx/app/edxapp/edxapp_env && cd /edx/src/sfe_parent/studio-frontend && export npm_config_prefix=/edx/src/sfe_parent/symlinks && npm link && rm -rf /edx/src/sfe_parent/studio-frontend/node_modules'
+	# step 2: install symlinked package under edx-platform
+	# if studio-frontend/node_modules exists during this step, it can cause issues
+	docker exec -t edx.devstack.studio bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && export npm_config_prefix=/edx/src/sfe_parent/symlinks && npm link @edx/studio-frontend'
+	# step 3: repair installations
+	docker exec -t edx.devstack.studio bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && npm update && npm install'
+	docker exec -t edx.devstack.studio bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform/node_modules/@edx/studio-frontend && npm install && npm run build'
+	# step 4: update assets properly, so webservers know about them
+	docker exec -t edx.devstack.studio bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && paver update_assets'
+
+devstack.update: ## moves newly webpack-generated assets to a place where studio can find them
+	docker exec -t edx.devstack.studio bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && paver webpack && python ./manage.py cms --settings=devstack_docker collectstatic --noinput'
```

Comment:
This won't be needed if the asset watchers are running; do we want to leave it in anyway for manual updates?

Reply:
Also I need to test this - I don't believe the final `collectstatic` is needed any longer.

Reply:
Confirmed - when the asset watchers are *not* running (`dev.up` rather than `dev.up.all`), the initial `source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && paver webpack` is all that's needed to force a cms update of your locally-installed `studio-frontend` code.

Reply:
Echoing Farhanah's comment this morning, did you get a chance to test this? What were the outcomes?